doc/CONFIGURE.md

```diff
@@ -0,0 +1,205 @@
+
+* **Name:** Name that uniquely identifies an agent within this Jenkins installation.
+* **Description:** Optional human-readable description for this agent. 
+* **\# of Executors:** The maximum number of concurrent builds that Jenkins may perform on this agent.
+* **Remote root directory:** An agent needs to have a directory dedicated to Jenkins. 
+Specify the path to this directory on the agent. It is best to use an absolute path, 
+such as /var/jenkins or c:\jenkins. This should be a path local to the agent machine.
+* **Labels:** Labels (or tags) are used to group multiple agents into one logical group. 
+You will use them to select it as agent for a build. Multiple labels must be separated by a space.
+* **Usage:** Controls how Jenkins schedules builds on this node.
+    * **Use this node as much as possible:** This is the default setting. In this mode, Jenkins uses this node freely.
+    Whenever there is a build that can be done by using this node, Jenkins will use it.
+    * **Only build jobs with label expressions matching this node:**
+    In this mode, Jenkins will only build a project on this node when that project is restricted to certain nodes using a label expression,
+    and that expression matches this node's name and/or labels.
+    This allows a node to be reserved for certain kinds of jobs. For example, if you have jobs that run performance tests,
+    you may want them to only run on a specially configured machine, while preventing all other jobs from using that machine.
+    To do so, you would restrict where the test jobs may run by giving them a label expression matching that machine. 
+    Furthermore, if you set the # of executors value to 1, you can ensure that only one performance test will execute at 
+    any given time on that machine; no other builds will interfere.
+* **Launch method:** Select *Launch agent agents via SSH* Starts a agent by sending commands over a secure SSH connection.
+
+![](images/ssh-node-basic-config.png)
+
+## Configure Launch agent agents via SSH
+
+Once you selected the **Launch method** to **Launch agent agents via SSH**, you can configure your SSH agent settings.
+
+### Required settings
+
+* **Host:** Hostname or IP of the agent, it should be resolved and reachable from the Jenkins instance. 
+* **Credentials:** Select the credentials to be used for logging in to the remote host. See [Integration with SSH Credentials Plugin](#integration-with-ssh-credentials-plugin)
+* **Host Key Verification Strategy:** Controls how Jenkins verifies the SSH key presented by the remote host whilst connecting. See [Host Key Verification Strategy](#host-key-verification-strategy)
+
+![](images/ssh-node-launcher-config.png)
+
+### Advanced settings
+
+* **Port:** The TCP port on which the slave's SSH daemon is listening, usually 22.
+* **JavaPath** This java Path will be used to start the jvm. (/mycustomjdkpath/bin/java ) If empty Jenkins will search java command in the agent.
```

Comment:
Path shouldn't be capitalized here. And JVM should be capitalized (I won't be a stickler about Java versus java since at least java is a real word besides being a programming language).